src/Sfa.Das.Sas.Indexer.AzureWorkerRole/app.config

```diff
@@ -59,6 +59,7 @@
     <add key="VstsGitFrameworksFolderPath" value="frameworks/local/json" />
     <add key="ImServiceBaseUrl" value="https://hub.fasst.org.uk" />
     <add key="ImServiceUrl" value="Learning%20Aims/Downloads/Pages/default.aspx" />
+    <add key="ImServiceLinkText" value="LARS CSV Version 5" />
     <!-- Elasticsearch -->
```

Comment:
innerhtml looks like this 
`innerHTML:"<span>LARS CSV V</span>ersion 5​"`

Do you think its better to use innerText in `AngleSharpService.cs line 50`?
innerText looks like this
`innerText:"LARS CSV Version 5​"`